dashboard/app/models/script_level.rb

```diff
@@ -421,6 +421,11 @@ def summarize(include_prev_next=true, for_edit: false)
       summary[:conceptDifficulty] = level.summarize_concept_difficulty
       summary[:assessment] = !!assessment
       summary[:challenge] = !!challenge
+
+      # TODO: (charlie) we will move this string conversion out of this
+      # `for_edit` block as soon as code studio updates to use strings for ids
+      summary[:ids] = summary[:ids].map(&:to_s)
+      summary[:activeId] = summary[:activeId].to_s
```

Comment:
I'm not sure I understand this comment -- to me, these lines already look like code studio using strings for IDs. does "code studio" refer to the client code? If so, that would help to clarify.

Reply:
the update for all our apps code to start using strings is in a separate PR that will be merged after this, so for now we only want ids to be strings in levelbuilder, which is why these `to_s` calls are in this `if for_edit` block. in my other PR i move them out of the `if for_edit` block so that all calls to this function will return strings. does that make sense?

Reply:
ok, if this comment will be going away soon then this sounds fine. :-) it sounds like you are referring to our frontend / client / js / apps code as "code studio". I had always thought of "code studio" as a synonym for dashboard or studio.code.org, frontend and backend. If I'm still misunderstanding please let me know, but I don't think it affects this review.

Reply:
yes, as soon as this PR gets merged i will be working on the followup PR. i referred to the other code as "code studio" because when i took this work over, Jessie told me she was done with all the code studio changes but hadn't handled levelbuilder yet, so i just rolled with that language. :)